ert_storage/database_schema.py

```diff
@@ -66,6 +76,17 @@ def record_type(self) -> RecordType:
     def record_type(self, other: RecordType) -> None:
         self._record_type = other.value
 
+    @property
+    def data(self) -> Any:
+        if self.record_type == RecordType.file:
+            return self.file.content
+        elif self.record_type == RecordType.float_vector:
+            return self.f64_matrix.content
+        else:
```

Comment:
Record should be able to be completely opaque as well.

Reply:
Yes. `RecordType.file`

Reply:
https://github.com/equinor/ert-storage/blob/070dad26534d1ce75a4b013f2be6080956cf603a/tests/integration/test_records.py#L185-L190